### PR TITLE
PHPdoc fix (add where was clear), some code style fixes, removed some comments from prod

### PR DESCRIPTION
**Pull request by tahrz**
… comments from prod

### CODE REVIEW

**Comment by skipperbent** [comment]
Nice work. One thing though, if you don't return static, the code completion won't recognize the new class if you extend it from a parent class; then only methods and properties from the base class will be available.

**Comment by Tahrz** [comment]
> Nice work. One thing though, if you don't return static, the code completion won't recognize the new class if you extend it from a parent class; then only methods and properties from the base class will be available.

Hi! Thanks, can you comment line from changed files, to show what exactly mean?

**Comment by skipperbent** [comment]
Sorry about that, ive added the explanation on a line so it hopefully makes more sense.

But nevertheless it's great work :)

**Comment by Tahrz** [comment]
> Sorry about that, ive added the explanation on a line so it hopefully makes more sense.
> 
> But nevertheless it's great work :)

:) sorry, I don't see your explanation on any line :(

File: src/Pecee/Http/Input/InputFile.php [hunk 1]
```diff
@@ -85,9 +85,8 @@ public function getSize(): string
     }
 
     /**
-     * Set file size
      * @param int $size
-     * @return static
+     * @return IInputItem
```
[the remark this answers is not in the document]
Comment (Tahrz) [reply src/Pecee/Http/Input/InputFile.php]: Oh, okay, thanks!

**Pull request closed without merging**